app/assets/stylesheets/legacy.scss

```diff
@@ -1157,11 +1157,11 @@ http://particletree.com/features/rediscovering-the-button-element/ */
 .widgets a, .widgets button {
     display:block;
     float: left;
-    margin:0px 7px 0 0;
+    margin:5px 10px 0 0;
     background-color:#f5f5f5;
-    border:1px solid #dedede;
-    border-top:1px solid #eee;
-    border-left:1px solid #eee;
+    border:1px solid #999;
+    border-top:1px solid #ccc;
+    border-left:1px solid #ccc;
```

Comment:
These changes to the margin and the border will have a global effect. If these are only for the reviewed button, could you please create a separate class for that and use these colors there?